lib/pages/chat/events/message_content.dart

```diff
@@ -144,33 +144,13 @@ class MessageContent extends StatelessWidget
                 !event.redacted &&
                 event.isRichMessage &&
                 containedLink.isEmpty) {
-              var html = event.formattedText;
-
-              if (event.messageType == MessageTypes.Emote) {
-                html = '* $html';
-              }
-              final bigEmotes = event.onlyEmotes &&
-                  event.numberEmotes > 0 &&
-                  event.numberEmotes <= 10;
               return Padding(
                 padding: MessageContentStyle.emojiPadding,
-                child: HtmlMessage(
-                  html: html,
-                  defaultTextStyle: Theme.of(context).textTheme.bodyLarge,
-                  linkStyle: TextStyle(
-                    color: Theme.of(context).colorScheme.secondary,
-                    decoration: TextDecoration.underline,
-                    decorationColor: textColor.withAlpha(150),
-                  ),
-                  room: event.room,
-                  emoteSize: bigEmotes ? fontSize * 3 : fontSize * 1.5,
-                  bottomWidgetSpan: Visibility(
-                    visible: false,
-                    maintainSize: true,
-                    maintainAnimation: true,
-                    maintainState: true,
-                    child: endOfBubbleWidget,
-                  ),
+                child: MessageTextContent(
+                  event: event,
+                  textColor: textColor,
+                  fontSize: fontSize,
+                  endOfBubbleWidget: endOfBubbleWidget,
```

Comment:
why we need to change widget for emoji?

Reply:
ah, got your idea. so we should change the name of widget

Reply:
done